pom.xml

```diff
@@ -173,6 +173,11 @@ THE SOFTWARE.
       <artifactId>wiremock</artifactId>
       <version>2.1.12</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
```

Comment:
Removed; using wiremock-standalone which will include all the pieces without conflicts.